website/src/categories/components/radio-group/usage.md

```diff
@@ -8,106 +8,126 @@ eleventyNavigation:
 <section class="no-heading">
 
```

Comment:
I'm not sure what's going on, but when I'm running the website locally everything looks fine, but here in the preview there is sth wrong with the radio component detail page:

<img width="2904" alt="image" src="https://github.com/sl-design-system/components/assets/111562742/af8e354f-2262-4597-b50a-b467741d009e">

Reply:
That is code from the `code` tab 🤔